src/main/java/pro/sky/java/course2/examinerservice/controller/ExamController.java

```diff
@@ -0,0 +1,25 @@
+package pro.sky.java.course2.examinerservice.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.java.course2.examinerservice.model.Question;
+import pro.sky.java.course2.examinerservice.service.ExaminerService;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class ExamController {
+    ExaminerService examinerService;
+
+    public ExamController(ExaminerService examinerService) {
+        this.examinerService = examinerService;
+    }
+
+    @GetMapping("/getQuestions")
```

Comment:
```suggestion
    @GetMapping
```
по заданию здесь нет ничего